apps/selvbetjening-ui/src/common/SystemUtilgjengelig.tsx

```diff
@@ -41,7 +41,7 @@ export const SystemUtilgjengelig = () => {
 
     return (
         <main>
-            <HStack justify="center" padding="8">
+            <HStack justify="center" padding="8" minHeight="calc(100vh - 158px)">
```

Comment:
Hardkodet `158px` kan potensielt skape mer jobb hvis det endrer seg og man da må oppdatere alle filene. Mulig å ha det i en const eller lignende for å kun trenge å oppdatere 1 sted.